docs/scripts/fledge_plugin_list

```diff
@@ -172,3 +269,36 @@ to extend the set of notification deliveries.
 
 EOF6
 table "$REPOSITORIES" "Fledge Notification Delivery Plugins" notify
+
+if [[ -d /tmp/keywords ]]; then
+	rm -rf /tmp/keywords
+fi
+cat >>$output << EOF7
+
+South Plugins Grouped by Category
```

Comment:
Category or Keyword?

Reply:
Keyword means less to users, hence category is a better title to use.